api.bs

```diff
@@ -1516,6 +1516,27 @@ this difference cannot be detected
 by the site receiving the conversion report.
 
 
+## Placing Identifiers in Filter Data ## {#privacy-filter-data}
+
+Because {{PrivateAttributionConversionOptions/filterData}}
+may contain arbitrary information, it is possible to place identifiers
+(e.g. of a user or an advertisement) in this field.
+It is important that identifiers in the filter data cannot be (mis)used
+for arbitrary tracking. The following measures mitigate this risk:
```

Comment:
```suggestion
The data obtained from attribution can use this data
when constructing a [=conversion report=],
which carries some risk of that identifying information
becoming available to the site that receives that report.
The following measures mitigate this risk:
```

Reply:
I'm having trouble parsing "The data obtained from attribution can use this data when constructing a [=conversion report=]".

How about:

> The attribution process can use this data when constructing a [=conversion report=], which implies some risk of that identifying information becoming available to the site that receives that report.